tree/ntuple/inc/ROOT/RNTupleReader.hxx

```diff
@@ -266,6 +266,8 @@ public:
    /// **Note**: if `T = void`, type checks are disabled. This is not really useful for this overload because
    /// RNTupleView<void> does not give access to the pointer. If required, it is possible to provide an `objPtr` of a
    /// dynamic type, for example via GetView(std::string_view, void *, std::string_view).
+   ///
+   /// Note that creating a View for a Streamer field is not supported.
```

Comment:
Is that a temporary limitation (need further development to support) or fundamental limitation (if it is fundamental, why?)

Reply:
As far as I understand it's something we could do, but at the cost of making the code more complex and I don't think there is a good incentive to do that (at least right now).

Reply:
Correct. We would need to "mark" these types as we do for collection proxies. In the case of streamer fields, the dictionary code could take care of the marking.